Library/Homebrew/diagnostic.rb

```diff
@@ -188,11 +188,14 @@ def __check_stray_files(dir, pattern, allow_list, message)
         files = Dir.chdir(dir) do
           (Dir.glob(pattern) - Dir.glob(allow_list))
             .select { |f| File.file?(f) && !File.symlink?(f) }
-            .map { |f| File.join(dir, f) }
+            .map do |f|
+              f.sub!(%r{/.*}, "/...") unless @verbose
```

Comment:
```suggestion
              f.sub!(%r{/.*}, "/*") unless @verbose
```
perhaps?

I don't typically see `...` used with filenames.